test/e2e/cypress/e2e/host_details.cy.js

```diff
@@ -363,6 +364,48 @@ context('Host Details', () => {
     });
   });
 
+  describe('Saptune Summary for this host should be displayed', () => {
+    const saptuneSummarySelector = '.pt-8';
+    it('should show not installed status', () => {
+      cy.loadScenario('host-vmhdbdev01-saptune-uninstalled');
+      cy.get(saptuneSummarySelector)
+        .should('contain', 'Saptune Summary')
+        .should('contain', 'Package')
+        .should('contain', 'Not installed')
+        .should('contain', 'Configured Version')
+        .should('contain', '-')
+        .should('contain', 'Tuning')
+        .should('contain', '-');
+    });
+
+    it('should show package version, configured version and tuning status', () => {
+      cy.loadScenario('host-vmhdbdev01-saptune-compliant');
+      cy.get(saptuneSummarySelector)
+        .should('contain', 'Saptune Summary')
+        .should('contain', 'Package')
+        .should('contain', '3.1.0')
+        .should('contain', 'Configured Version')
+        .should('contain', '3')
+        .should('contain', 'Tuning')
+        .should('contain', 'Compliant');
+    });
+
+    it('should show version is not supported status', () => {
+      const versionStatusIcon =
+        '.grid-rows-2 > :nth-child(1) > :nth-child(2) > :nth-child(1)';
+      cy.loadScenario('host-vmhdbdev01-saptune-unsupported');
+      cy.get(saptuneSummarySelector)
+        .should('contain', 'Saptune Summary')
+        .should('contain', 'Package')
+        .should('contain', '3.0.0')
+        .should('contain', 'Configured Version')
+        .should('contain', '-')
+        .should('contain', 'Tuning')
+        .should('contain', '-');
+     // cy.get(versionStatusIcon).should('have.class', 'fill-yellow-500');
```

Comment:
We don't need to tests the icons here. It is already tested in the UT. With the text is enough